js/companies.js

```diff
@@ -68,5 +104,8 @@ chrome.runtime.onMessage.addListener((req, sender, res) => {
     if (req.getTopBlocked) {
         res(Companies.getTopBlocked(req.getTopBlocked))
     }
+    else if (req.getTopBlockedByPages) {
+        res(Companies.getTopBlockedByPages(req.getTopBlockedByPages))
+    }
     return true;
```

Comment:
It's necessary. Buried in the api docs is a note that if you don't return true the message pipe is closed instead of waiting for you to send a response. https://developer.chrome.com/extensions/runtime#event-onMessage

Reply:
Ok